core/src/test/scala/org/broadinstitute/dsde/rawls/RawlsTestUtils.scala

```diff
@@ -71,8 +71,8 @@ trait RawlsTestUtils extends Suite with TestDriverComponent with Matchers {
   }
 
   // MockServer's .withBody doesn't have a built-in string contains feature.  This serves that purpose.
-  def mockServerContains(text: String): StringBody = {
-    val anythingWithNewlines = "((.|\n|\r)*)"
-    StringBody.regex(anythingWithNewlines + Regex.quote(text.toString) + anythingWithNewlines)
+  def mockServerContains(text: String): RegexBody = {
+    // "(?s)" turns on DOTALL mode, where a "." matches a line break as well as any character
+    new RegexBody("(?s).*" + Regex.quote(text) + ".*")
```

Comment:
why did this have to change? Was there a java 8 -> compatibility issue here?

Reply:
The new version of mockserver doesn't support `Stringbody.regex`, it has `RegexBody` instead.